feature/staticroute/TwoPortConfig/static_route_test.go

```diff
@@ -0,0 +1,101 @@
+package gnmi_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func TestGNMI(t *testing.T) {
+
+	// Configure a DUT
+
+	dut := ondatra.DUT(t, "dut")
+	dp1 := dut.Port(t, "port1")
+	dp2 := dut.Port(t, "port2")
+
+	ifCfg1 := &telemetry.Interface{
+		Name:        ygot.String(dp1.Name()),
+		Description: ygot.String("From Ixia"),
+	}
+
+	ifCfg1.GetOrCreateSubinterface(0).
+		GetOrCreateIpv4().
+		GetOrCreateAddress("192.0.2.10").
+		PrefixLength = ygot.Uint8(31)
+
+	ifCfg2 := &telemetry.Interface{
+		Name:        ygot.String(dp2.Name()),
+		Description: ygot.String("To Ixia"),
+	}
+
+	ifCfg2.GetOrCreateSubinterface(0).
+		GetOrCreateIpv4().
+		GetOrCreateAddress("192.0.2.20").
+		PrefixLength = ygot.Uint8(31)
+
+	dut.Config().Interface(dp1.Name()).Update(t, ifCfg1)
+	dut.Config().Interface(dp2.Name()).Update(t, ifCfg2)
+
+	Ni1 := &telemetry.NetworkInstance{}
+
+	Ni1.GetOrCreateProtocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "static").
+		GetOrCreateStatic("10.0.0.0/24").
+		GetOrCreateNextHop("h1").NextHop = telemetry.UnionString("192.0.2.10")
```

Comment:
Our flow goes to ATE port 1 on line 78 - but we might want to check whether this can be forwarded to ATE port 2.

One way that we might make this clearer is we said:

```golang

atePort2 := &Attributes{
   IPv4: "192.0.2.xx",
}

...NextHop = telemetry.UnionString(atePort2.IPv4)
```

Reply:
But now we lost our looping...

```golang
atePort1 := &Attributes{ ... }
atePort2 := &Attributes{ ... }

atePorts := []*Attributes{atePort1, atePort2}
for _, port := range atePorts { ... }
```